cmd/koord-scheduler/app/server.go

```diff
@@ -196,6 +197,15 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	cc.InformerFactory.WaitForCacheSync(ctx.Done())
 	cc.KoordinatorSharedInformerFactory.WaitForCacheSync(ctx.Done())
 	cc.NRTSharedInformerFactory.WaitForCacheSync(ctx.Done())
+	// after all caches to sync
+	// we start to recover GangCache
+	gangCacheRaw := extHandle.GetGangCache()
+	gangCache, ok := gangCacheRaw.(*gang.GangCache)
+	if !ok {
+		klog.Errorf("Run scheduler get gangCache assert failed")
+	} else {
+		gangCache.RecoverGangCache()
```

Comment:
I think there should be no need to do this. You can generate and start the Informer in the plugin. This is reliable even when relying on PodInformer.

Reply:
got it, solved